app/src/main/java/com/android/sample/model/ingredient/Ingredient.kt

```diff
@@ -1,3 +1,16 @@
 package com.android.sample.model.ingredient
 
-class Ingredient(val barCode: Long, val name: String) {}
+data class Ingredient(
+    val barCode: Long? = null,
+    val name: String? = null,
+    val selectedImages: SelectedImages? = null,
+    val brands: String? = null
+)
+
+data class SelectedImages(val front: ImageUrls? = null)
+
+data class ImageUrls(
+    val display: String? = null,
+    val small: String? = null,
+    val thumb: String? = null
+)
```

Comment:
I don't think we should add the image urls as we're still working on the image database. This will lead to some temporary code, that will be removed later for the database implementation.